scripts/orca_json_to_qm9_npz.py

```
@@ -0,0 +1,97 @@
+        if "CARTESIAN COORDINATES (ANGSTROEM)" in line:
+            start_coords = True
+            continue
+        if start_coords and "---" in line:
+            start_coords = False
+            break
+        if start_coords and line.strip():
+            parts = line.split()
+            try:
+                Z.append(int(get_atomic_number(parts[0])))
+                R.append([float(parts[1]), float(parts[2]), float(parts[3])])
+            except:
```

Ruff (0.11.9)

Do not use bare `except`

(E722)